pkg/provisioner/ironic/ironic_test.go

```diff
@@ -62,3 +70,6 @@ func makeHost() *metal3v1alpha1.BareMetalHost {
 		},
 	}
 }
+
+// Implements provisioner.EventPublisher to swallow events for tests.
+func nullEventPublisher(reason, message string) {}
```

Comment:
Nice one, we could introduce a set of different hard-coded publishers (ie null, std) to be reused in all the tests to remove some other lines of code